pennylane/drawer/tape_mpl.py

```diff
@@ -450,14 +452,31 @@ def tape_mpl(
 
     .. code-block:: python
 
-        fig, ax = tape_mpl(tape, wire_options={'color':'teal', 'linewidth': 5},
+        fig, ax = qml.drawer.tape_mpl(tape, wire_options={'color':'teal', 'linewidth': 5},
                     label_options={'size': 20})
 
     .. figure:: ../../_static/tape_mpl/wires_labels.png
             :align: center
             :width: 60%
             :target: javascript:void(0);
 
+    Additionally, ``wire_options`` may contain wire labels as keys and dictionaries with
```

Comment:
Same here.

Reply:
Thanks a lot for the comment, I have updated the necessary function description part and the mpldrawer.py 👍 . I am not sure why some checks failed this time though 😢 .

Reply:
It is a problem with the CI pipeline, not on you! Let's retrigger CI and see whether it passes.